src/test/java/uk/gov/hmcts/reform/civil/handler/callback/user/CreateClaimCallbackHandlerTest.java

```diff
@@ -1173,7 +1173,6 @@ void shouldNotCopyRespondent1OrgPolicyDetailsFor1v2SameUnregisteredSolicitorScen
 
         @Test
         void shouldAssignCaseName1v2_whenCaseIs1v2GlobalSearchEnabled() {
```

Comment:
Please rename test as globalSearch int name is no longer relevant
ie shouldAssignCaseName1v2_whenCaseIs1v2()

Reply:
Test name updated